price_feeds_poller.py

```diff
@@ -362,7 +375,7 @@ def log_loop(
           # If no update is pending:
           else :
             
-            if elapsed_secs >= pf["cooldown"] - network_witnet_resolution_secs:
+            if elapsed_secs >= pf["cooldown"] - network_witnet_resolution_secs - network_evm_finalization_secs:
```

Comment:
Since `network_witnet_resolution_secs` has been replaced with `network_witnet_resolution_secs + network_evm_finalization_secs`, maybe it's better to define a new variable like `total_finalization_secs = network_witnet_resolution_secs + network_evm_finalization_secs` in main. This way you avoid having to handle these variables separately.